rules_service/settings/base.py

```diff
@@ -29,6 +29,11 @@
     '.app.cloud.gov',
 ]
 
+CORS_ORIGIN_REGEX_WHITELIST = (
+    'localhost:(\d{4})'
```

Comment:
Is the port number significant in this? If it is, perhaps the 2nd one should also have a regex like the 1st one, ie, `\d{4}`?

Reply:
Not entirely, but if, for example, an app is already running on the default port `webpack` uses for its dev server (3000), it will automatically try and use port 3001.

Seemed like a useful thing to have the ports be dynamic as well.

I do realize the ports can go higher than 4 digits, but I'm not sure what a sensible cutoff would be.

We could (should?) probably do the same to the IP for consistency.